src/main/java/betterquesting/api2/client/gui/panels/lists/CanvasScrolling.java

```diff
@@ -94,30 +97,24 @@ public CanvasScrolling enableBlocking(boolean state) {
         return this;
     }
 
-    public IGuiRect getScrollBounds() {
-        return this.scrollBounds;
-    }
+    public IGuiRect getScrollBounds() { return this.scrollBounds; }
```

Comment:
the standard formatting appears to include a new line between `{`, `}`, and the statement, as it was before this change. since IDEA will automatically do this shortening, theres no real benefit to doing it manually. i think this part should be reverted